src/explore-education-statistics-admin/src/pages/users/__tests__/UserInvitePage.test.tsx

```diff
@@ -447,9 +476,5 @@ describe('UserInvitePage', () => {
         </TestConfigContextProvider>
       </MemoryRouter>,
     );
-
-    expect(
-      await screen.findByText('Manage access to this service'),
-    ).toBeInTheDocument();
```

Comment:
You get stung by missing that this was here in one of the render methods doing this?

I agree with moving it out - even if it's more lines I wouldn't expect the render method to be doing something like this.